x-pack/test/functional/services/ml/job_table.ts

```diff
@@ -48,8 +54,25 @@ export function MachineLearningJobTableProvider(
   const headerPage = getPageObject('header');
   const testSubjects = getService('testSubjects');
   const retry = getService('retry');
+  const find = getService('find');
 
   return new (class MlJobTable {
+    public async assertJobsInTable(expectedJobIds: string[]) {
+      for await (const expectedId of expectedJobIds) {
+        const visibleText = await testSubjects.getVisibleText('mlJobListColumnId');
+        expect(visibleText).to.be(expectedId);
+      }
+    }
```

Comment:
I think this assertion wouldn't work if the table has multiple rows. I'd suggest to use `parseJobTable`, then get all the job ids and compare them with the expected ids (through sorting and exact match like we do in other places).

Reply:
https://github.com/elastic/kibana/pull/179321/commits/f9fdc495f4b8a6c402567294dc91621c599c065a